### DISCUSSION TITLE
Small phone is best phone - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#30] [General]
# Small phone is best phone - Alan Pope's blog

I’ve owned plenty of mobile phones over the years. My current daily driver is iPhone 13 Mini, sporting a bright, funky, and cheap OIIAEE Silicone Case. Look at it. It’s gorgeous (ignore the notification badge count, please).
Picture taken with the potato camera on my old OnePlus 5.
As I mentioned in a previous post, my favourite phones include the flyweight Nokia 6600 (2.1"), bantamweight (2.4") and featherweight iPhone 4s (3.

https://popey.com/blog/2023/09/small-phone-is-best-phone/

**Comment by skellat** [comment]
I am at the point where my phone needs are getting weird.  The main reason that I really, really should break down and get an iPhone 15 is that I need satellite communications.  My round-trip commute to the college to teach takes me through places that have as much cell coverage as Jupiter and Mars have.  That is to say, none at all.  If I have car trouble while driving in these dead zones it can be a long walk in the cold to get help in the winter.

I *could* get an ACR Bivy Stick and mate that with my iPhone SE 2 to get satcoms too.  It wouldn’t have the SOS & AAA integration that Apple has worked out.  It cost 25-30% the price of an iPhone 15 to just pick up a Bivy Stick and you still have a Monthly Recurring Charge to pay for its service.

As much as I might want a small phone too it doesn’t seem in the cards for me.

**Comment by Hund** [comment]
What's wrong with Android? :)
